### PR TITLE
8896 standalone remote trial session form UI

### PR DESCRIPTION
**Pull request by mrinsin**
image

image

### CODE REVIEW

File: web-client/src/ustc-ui/DateInput/DateInput.jsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const DateInput = props => {
       className={className}
       errorText={errorText}
       hideLegend={hideLegend}
+      hintText={hintText}
```
Comment (rachelschneiderman) [inline web-client/src/ustc-ui/DateInput/DateInput.jsx:28]: Nit: Is this default value necessary?

File: web-client/src/styles/custom.scss [hunk 2]
```diff
@@ -462,6 +462,10 @@ select[disabled] {
   background: color($theme-color-warning-lighter);
 }
 
+.negative-margin-top {
```
Comment (rachelschneiderman) [inline web-client/src/styles/custom.scss]: Nit: I don't think we are using this anymore?

Comment (mrinsin) [reply web-client/src/styles/custom.scss:465]: Guess not! TY!!

File: web-client/src/ustc-ui/DateInput/DateInput.jsx (hunk 1 above)
Comment (mrinsin) [reply web-client/src/ustc-ui/DateInput/DateInput.jsx:28]: nope its not! TY!

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**